### PR TITLE
Add API section for DSTU 2 with Conformance and Patient Resources

### PR DESCRIPTION
**Pull request by jfrhong**

### CODE REVIEW

File: content/dstu2.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+## Pagination
+
+The pagination links are included in the Bundle when a resource returns multiple items. It is important to
+follow these Link header values instead of constructing your own URLs.
+
+    {
+        "resourceType": "Bundle",
+        "id": "962a50b1-a367-475f-b9c9-810c1b3c6b35",
+        "base": https://fhir.sandboxcernerpowerchart.com/dstu2/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f,
```
Comment (cerbs) [inline content/dstu2.md]: Need quotes around URL

File: layouts/dstu1_api.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <%= render 'head' %>
 
-<body class="api">
+<body class="api dstu1_api">
```
Comment (dennispatterson) [inline layouts/dstu1_api.html]: Just use the dstu-specific class since that's what has representation in the css.

File: lib/resources.rb [hunk 3]
```diff
@@ -872,6 +874,337 @@ def get_resource(key)
...
+                        "text"=> "Married"},
+      "active"=> true
+    }
+
+    DSTU2_PATIENT_BUNDLE ||={
+      "resourceType"=> "Bundle",
+      "id"=> "60eb43c1-1258-4d30-9d9e-ed0b9e2cc33b",
+      "type"=> "searchset",
+      "base"=> "https://fhir.sandboxcernerpowerchart.com/dstu2/open/d075cf8b-3261-481d-97e5-ba6c48d3b41f",
+      "total"=> 15,
```
Comment (dennispatterson) [inline lib/resources.rb]: This should be 2 for the sake of the example

**Comment by cerbs** [comment]
+1

**Comment by dennispatterson** [comment]
:+1:  assuming squashed commits

**Comment by MaxPhilips** [comment]
+1 after squash

**Comment by naveenaluru** [comment]
:+1:

**Pull request merged**